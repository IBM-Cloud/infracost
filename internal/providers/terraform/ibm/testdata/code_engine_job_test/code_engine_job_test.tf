terraform {
  required_providers {
    ibm = {
      source = "IBM-Cloud/ibm"
    }
  }
}

provider "ibm" {
  region = "us-south"
}

resource "ibm_resource_group" "test_group" {
  name = "test-resource-group"
}

resource "ibm_code_engine_project" "ce_project" {
  name              = "ce_project"
  resource_group_id = ibm_resource_group.test_group.id
}

resource "ibm_code_engine_job" "ce_job" {
  project_id         = ibm_code_engine_project.ce_project.id
  name               = "ce-job"
  image_reference    = "icr.io/codeengine/helloworld"
  scale_memory_limit = "4G"
  scale_cpu_limit    = "1"
}

resource "ibm_code_engine_job" "ce_job2" {
  project_id         = ibm_code_engine_project.ce_project.id
  name               = "ce-job2"
  image_reference    = "icr.io/codeengine/helloworld"
  scale_memory_limit = "4G"
  scale_cpu_limit    = "1"
}
