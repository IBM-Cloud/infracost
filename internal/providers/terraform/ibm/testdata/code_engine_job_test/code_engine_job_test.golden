
 Name                              Monthly Qty  Unit        Monthly Cost 
                                                                         
 ibm_code_engine_job.ce_job                                              
 ├─ Virtual Processor Cores Hours            1  vCPU Hours         $0.12 
 └─ RAM Hours                                4  GB Hours           $0.05 
                                                                         
 ibm_code_engine_job.ce_job2                                             
 ├─ Virtual Processor Cores Hours            1  vCPU Hours         $0.12 
 └─ RAM Hours                                4  GB Hours           $0.05 
                                                                         
 OVERALL TOTAL                                                     $0.35 
──────────────────────────────────
4 cloud resources were detected:
∙ 2 were estimated
∙ 2 were free:
  ∙ 1 x ibm_code_engine_project
  ∙ 1 x ibm_resource_group